src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -224,14 +225,25 @@ Copyright (c) .NET Foundation. All rights reserved.
     </RemoveDuplicates>
 
     <!-- Remove projects that do not support restore. -->
-    <MsBuild
+    <!-- With SkipNonexistentTargets support -->
+    <MsBuild Condition=" '$(RestoreUseSkipNonexistentTargets)' == 'true' "
+        BuildInParallel="$(RestoreBuildInParallel)"
         Projects="@(FilteredRestoreGraphProjectInputItemsWithoutDuplicates)"
         Targets="_IsProjectRestoreSupported"
-        ContinueOnError="$(RestoreContinueOnError)"
-        Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration);
-                    %(_MSBuildProjectReferenceExistent.SetPlatform);
-                    $(_GenerateRestoreGraphProjectEntryInputProperties)"
-        RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)">
+        SkipNonexistentTargets="true"
```

Comment:
@emgarten 
Any reason fro not adding SkipNonexistentProjects="true" in this call?